libs/utils/metrics.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+
+# TODO: In the future maybe make sense allow the users to configure this parameter
+PROCESSING_COUNT_THRESHOLD = 2
+
+
+def process_distance_violation_for_object(distance_violations):
+    # TODO: This is the first version of the metrics and is implemented to feed the current dashboard.
+    # When we define the new metrics we will need to change that logic
+    object_detections = 0
+    object_violations = 0
+    current_status = None
+    processing_status = None
+    processing_count = 0
+
+    for dist_violation in distance_violations:
+        if processing_status != dist_violation:
+            processing_status = dist_violation
+            processing_count = 0
+        processing_count += 1
+        if current_status != processing_status and processing_count > PROCESSING_COUNT_THRESHOLD:
```

Comment:
Personally I think it is clearer if you use a `>= PROCESSING_COUNT_THRESHOLD` and set the threshold to 3

Reply:
Done